Source/Android/app/src/main/res/layout/fragment_cheat_warning.xml

```diff
@@ -23,9 +12,20 @@
         android:layout_margin="@dimen/spacing_small"
         android:text="@string/cheats_open_settings"
         android:textColor="@color/dolphin_white"
-        app:layout_constraintStart_toEndOf="@id/text_warning"
+        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent" />
+        app:layout_constraintStart_toEndOf="@id/text_warning"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/text_warning"
+        android:layout_width="0dp"
+        android:layout_height="match_parent"
+        android:layout_margin="@dimen/spacing_large"
+        android:text="@string/cheats_disabled_warning"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@id/button_settings"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
```

Comment:
Any particular reason why the two UI elements were reordered in the XML file?

Reply:
Android Studio reordered these automatically. I reverted the ordering to the original state.